source/Calamari.Tests/Fixtures/Retention/RetentionPolicyFixture.cs

```diff
@@ -307,23 +302,6 @@ public void ShouldNotDeleteAnythingWhenSpecifiedNumberOfSuccessfulDeploymentsNot
             deploymentJournal.Received().RemoveJournalEntries(Arg.Is<IEnumerable<string>>(ids => !ids.Any()));
         }
 
-        [Test]
-        public void ShouldApplyRetentionToMultiplePackages()
```

Comment:
The purpose of this test was to check that if the same package was in separate deployments, that it wouldn't be deleted as part of one deployment getting cleaned up. Since lifecycle retention policy should no longer be dealing with packages at all I've removed this test. The last part where it checks for an older entry being removed is already part of some of the above tests.